components/azure-iot/sdk_tests/tlsio_openssl_compact_ut/tlsio_openssl_compact_ut.c

```diff
@@ -0,0 +1,1616 @@
+// Copyright (c) Microsoft. All rights reserved.
+// Licensed under the MIT license. See LICENSE file in the project root for full license information.
```

Comment:
I see a fair number of unit test being added here. These tests don't seem to use the test framework which other ESP-IDF components use (description here: http://esp-idf.readthedocs.io/en/latest/api-guides/unit-tests.html). If this component is moved into ESP-IDF with the intended purpose of being maintained by Espressif, it would be great if we could run these unit tests. We can run the unit tests in our CI environment, but to do that, the tests need to use the above mentioned framework.